src/implementations/twig/components/footer-standardised/footer-standardised.story.js

```diff
@@ -1,190 +1,140 @@
-            );
-            if (linkItem.icon) {
-              linkItem.icon.name = text(
-                `sections[${i}][${j}].links[${k}].icon.name`,
-                linkItem.icon.name,
-                tabLabels.optional
-              );
-              linkItem.icon.path = text(
-                `sections[${i}][${j}].links[${k}].icon.path`,
-                defaultSprite,
-                tabLabels.optional
-              );
-              linkItem.icon.size = text(
-                `sections[${i}][${j}].links[${k}].icon.size`,
-                linkItem.icon.size,
-                tabLabels.optional
-              );
-            }
-          });
-        }
-      });
-    }
-  });
+const getArgTypes = (system) => {
+  const argTypes = {};
+  argTypes.hide_contact = {
+    name: system === 'EU' ? 'contact site name' : 'contact us',
+    type: { name: 'boolean' },
+    defaultValue: false,
+    description:
+      system === 'EU'
+        ? 'Hide "Contact site name" section'
+        : 'Hide "Contact us" section',
+    table: {
+      type: { summary: 'boolean' },
+      defaultValue: { summary: false },
+      category: 'States',
+    },
+  };
 
-  getExtraKnobs(data);
+  argTypes.hide_follow = {
+    name: 'follow us',
+    type: { name: 'boolean' },
+    defaultValue: false,
+    description: 'Hide "Follow us" section',
+    table: {
+      type: { summary: 'boolean' },
+      defaultValue: { summary: false },
+      category: 'States',
+    },
+  };
 
-  // Return the full specs.
-  return data;
+  if (system !== 'EU') {
+    argTypes.hide_about = {
+      name: 'about us',
+      type: { name: 'boolean' },
+      defaultValue: false,
+      description: 'Hide "About us" section',
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: false },
+        category: 'States',
+      },
+    };
+  }
+
+  argTypes.hide_relate_site = {
+    name: system === 'EU' ? 'optional links' : 'related sites',
+    type: { name: 'boolean' },
+    defaultValue: false,
+    description:
+      system === 'EU'
+        ? 'Hide "Optional links" section'
+        : 'Hide "Related sites" section',
+    table: {
+      type: { summary: 'boolean' },
+      defaultValue: { summary: false },
+      category: 'States',
+    },
+  };
+
+  if (system !== 'EU') {
+    argTypes.hide_class_name = {
+      name: 'class name',
+      type: { name: 'boolean' },
+      defaultValue: false,
+      description: 'hide "Class name" section',
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: false },
+        category: 'States',
+      },
+    };
+  }
+
+  return argTypes;
+};
+
+const prepareData = (data, args, system) => {
+  correctSvgPath(data);
+
+  if (data.rows[1][0][0].logo) {
+    data.rows[1][0][0].logo.src_mobile = logoEuMobile;
+    data.rows[1][0][0].logo.src_desktop = logoEuDesktop;
+  }
+
+  const res = JSON.parse(JSON.stringify(data));
+  if (args.hide_contact === true) {
```

Comment:
I love the solution overall, but why checking for `=== true`?
Wouldn't be enough to say `if (args.hide_contact)`..?